diff --git a/node_modules/connect-dynamodb/lib/connect-dynamodb.js b/node_modules/connect-dynamodb/lib/connect-dynamodb.js
index fce270c..c0ad8d2 100644
--- a/node_modules/connect-dynamodb/lib/connect-dynamodb.js
+++ b/node_modules/connect-dynamodb/lib/connect-dynamodb.js
@@ -156,12 +156,19 @@ module.exports = function (connect) {
           })
         )
         .then((result) => {
+          console.time(".then")
+          console.log(result)
           try {
+            console.time("getParsedSession")
             const sess = this.getParsedSession(result);
+            console.timeEnd("getParsedSession")
+            console.time("express-function")
             fn(null, sess);
+            console.timeEnd("express-function")
           } catch (err) {
             fn(err);
           }
+          console.timeEnd(".then")
         })
         .catch(fn);
     });
